GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -2281,9 +2283,18 @@ private void CommitMessageToolStripMenuItemDropDownOpening(object sender, EventA
         {
             var msg = AppSettings.LastCommitMessage;
             var maxCount = AppSettings.CommitDialogNumberOfPreviousMessages;
+            var authorPattern = string.Empty;
 
-            var prevMessages = Module.GetPreviousCommitMessages(maxCount)
+            if (ShowOnlyMyMessagesToolStripMenuItem.Checked)
+            {
+                var userName = Module.GetEffectiveSetting(SettingKeyString.UserName);
+                var userEmail = Module.GetEffectiveSetting(SettingKeyString.UserEmail);
+                authorPattern = $"^{userName} <{userEmail}>$";
```

Comment:
If this is a regex, at least dots should be escaped, shouldn't they?

Reply:
Yeah